client/components/ProductOverview/ImageGallery.jsx

```diff
@@ -72,8 +72,8 @@ const ImageGallery = ({
         />
       </div>
       <ThumbnailView photos={photos} handleClick={handleThumbNailClick} className="thumbnailComponent" currentIndex={imageIndex} />
-      {imageIndex === 0 ? null : <AiOutlineArrowLeft className="leftArrow" onClick={() => handleArrow('left')} size={arrowSize} />}
-      {imageIndex === length - 1 ? null : <AiOutlineArrowRight className="rightArrow" onClick={() => handleArrow('right')} size={arrowSize} />}
+      {imageIndex === 0 ? <AiOutlineArrowLeft className="leftArrow" onClick={() => handleArrow('left')} size={arrowSize} style={{ visibility: 'hidden' }} /> : <AiOutlineArrowLeft className="leftArrow" onClick={() => handleArrow('left')} size={arrowSize} />}
```

Comment:
Nice. 👌 These lines are getting pretty long and unwieldy. Would it be possible to break them down a bit?

Reply:
Good call, don't know what I was thinking. I have since moved the ternary operator inside of the style attribute as that was the only thing I needed to change.